libs/sdk-ui/src/base/vis/drilling.ts

```diff
@@ -145,7 +145,7 @@ function convertToEmpty(attributeItem: IResultAttributeHeader) {
         attributeHeaderItem: {
             ...attributeHeaderItem,
             // Send empty string for not, need to be updated for NULL in future
-            name: isEmpty ? "" : attributeHeaderItem.name,
+            name: isEmpty ? "" : attributeHeaderItem.name ?? "",
```

Comment:
please add brackets to improve readability

Reply:
Unfortunately Prettier strips those.